apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -131,6 +131,14 @@ type ContourSettings struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=42
 	WatchNamespaces []contour_api_v1.Namespace `json:"watchNamespaces,omitempty"`
+
+	// DisabledFeatures defines an array of Gateway API CRDs that will be ignored by
```

Comment:
Technically they are not all Gateway API resources (`extensionservices` is a Contour resource):
```suggestion
	// DisabledFeatures defines an array of resources that will be ignored by
```